.github/workflows/deploy-distributables.yml

```diff
@@ -1,16 +1,42 @@
-# When a new Release is created, deploy relevant
+# When a new release is created, deploy relevant
 # files to each of the generated repos.
-name: Deploy Framework
+name: Deploy Distributable Repos
 
 on:
   release:
     types: [published]
 
 jobs:
+  check-version:
+    name: Check for updated version
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # fetch all tags
+
+      - name: Get latest version
+        run: |
+          echo 'LATEST_VERSION<<EOF' >> $GITHUB_ENV
+          echo $(git describe --tags --abbrev=0) | sed "s/v//" >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+
+      - name: Search for updated version
+        if: ${{ env.LATEST_VERSION }}
+        run: |
+          if ! grep -rq system/CodeIgniter.php user_guide_src/source/conf.py -e "${{ env.LATEST_VERSION }}"; then
```

Comment:
1. Why `-r`?
2. Shouldn't we check both `system/CodeIgniter.php` and`user_guide_src/source/conf.py` have the latest version? Now it checks `system/CodeIgniter.php` or`user_guide_src/source/conf.py`.

Reply:
> 1. Why `-r`?

That `-r` is not needed. I'll remove.

> 2. Shouldn't we check both `system/CodeIgniter.php` and`user_guide_src/source/conf.py` have the latest version? Now it checks `system/CodeIgniter.php` or`user_guide_src/source/conf.py`.

Oh, you're right. I thought it checks for both presence.